tests/foreman/virtwho/ui/test_esx.py

```diff
@@ -54,6 +57,14 @@ def form_data():
 
 
 class TestVirtwhoConfigforEsx:
+    @pytest.mark.tier2
+    def setup_method(self):
```

Comment:
I would recommend you to use autouse pytest fixture instead of setup_method. The reason is in the next comment.

Reply:
> I would recommend you to use autouse pytest fixture instead of setup_method. The reason is in the next comment.

@jyejare Thank you very much for your suggestion, have already use fixture to realize this function, please review it ,many thanks